chain/client/src/client_actor.rs

```diff
@@ -1336,6 +1336,7 @@ impl ClientActorInner {
         };
 
         // If we produced the block, send it out before we apply the block.
+        self.client.chain.blocks_delay_tracker.mark_block_received(&block);
```

Comment:
What do you think about updating the comment(s) in `BlocksDelayTracker` , to clarify that "blocks received" are not only blocks received by other nodes, but also blocks self-produced?

Reply:
Good idea, implemented test_not_process_same_block_twice for that.